src/pages/filter/ui/query-builder/groups/chart/pie-chart/pie-chart.tsx

```diff
@@ -51,22 +55,30 @@ export const PieChart = ({ data }: IPieChartProps): ReactElement | null => {
           const optionPercentage = ((value / totalCountsOnChart) * 100).toFixed(
             2,
           )
+          const varaintsQuantityContent = t('filter.chart.variants', {
```

Comment:
I think, we can implement this with pure css flexbox layout. 
Please, try to use `flex-wrap: wrap` and `justify-content: space-between`

Reply:
I changed the implementation

Reply:
I changed the implementation